marketplace/netlify-build/README.md

```diff
@@ -0,0 +1,25 @@
+# Netlify Build and Preview
+
+This sidebar extension allows you to build and preview up to 3 Netlify sites.
+
+This is an alternative approach to using a webhook to trigger a build. The advantage
+is that the editor gets control on when a build is executed instead of responding
+to automatic events such as `Entry.publish`.
+
+![Netlify Sidebar UI Sample](./assets/netlify-triggering.gif)
```

Comment:
Can we have the full flow in the gif? "Triggering, building, last built by..."

Reply:
Done.